docs/UserGuide.md

```diff
@@ -117,7 +125,14 @@ Format: `deallocate si/<student_index>`
 * The student at `student_index` must have been allocated a room. Otherwise, an error message is displayed.
 
 Examples:
-* `deallocate si/1` deallocates the room for the student at `student_index` 1. 
+* `deallocate si/1` deallocates the room for the student at `student_index` 1.
+
+##### Before deallocation
+Refer to "After allocation" above.
+
+##### After deallocation
+
+<img src="https://user-images.githubusercontent.com/27071473/97580237-0040da00-1a2e-11eb-9337-f3bf29f85bed.png">
```

Comment:
Bro put the image in our `docs/images` folder and use the markdown format to insert it.

Reply:
Bro it's "change(s)" requested not change**d** requested.

Reply:
Changed maded